src/ngraph/runtime/cpu/cpu_tensor_view.cpp

```diff
@@ -72,9 +78,19 @@ runtime::cpu::CPUTensorView::CPUTensorView(const ngraph::element::Type& element_
     }
 }
 
+runtime::cpu::CPUTensorView::CPUTensorView(const ngraph::element::Type& element_type,
+                                           const Shape& shape,
+                                           const string& name)
+    : CPUTensorView(element_type, shape, nullptr, name)
+{
+}
+
 runtime::cpu::CPUTensorView::~CPUTensorView()
 {
-    free(buffer);
+    if (buffer != nullptr)
```

Comment:
This check should be removed

Reply:
will do; just curious, is it because it is unnecessary?  this check is used in interpreter before this PR, and it was missing here.

Reply:
It wasn't here because it's unnecessary, yes